datahub-web-react/src/app/entity/shared/containers/profile/header/EntityHeader.tsx

```diff
@@ -143,6 +154,9 @@ export const EntityHeader = ({ refreshBrowser, headerDropdownItems, isNameEditab
                                 </ExternalUrlButton>
                             </ExternalUrlContainer>
                         )}
+                        {/* headerActionItems && (
+                            <EntityActions actionItems={headerActionItems} refetchForEntity={refetch} />
+                        ) */}
```

Comment:
^

Reply:
Intentional - After this PR the actions will remain hidden until the backend is implemented!